src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String size;
```

Comment:
Looks strange when you declare variable "size" by String. If we want use let`s say size like SMALL, BIG etc, better to use ENUM

Reply:
I don't know why it was necessary to invent these fields, so I inserted random types)))